packages/event-log-explorer.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,13 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'Event Log Explorer'
```

Comment:
```suggestion 
$toolName = 'elex'
```
I think the tool name is `elex` (usually the file name with the `.exe`)

Reply:
I don't think so, or I would have to change from

```powershell
$toolDir = Join-Path ${Env:ProgramFiles(x86)} $toolName
$executablePath = Join-Path $toolDir "elex.exe"
```

to

```powershell
$toolDir = Join-Path ${Env:ProgramFiles(x86)} "Event Log Explorer"
$executablePath = Join-Path $toolDir "${toolName}.exe"
``` 

In addition, `lnk` will be `elex` instead of `Event Log Explorer`.

Reply:
From our [issue template](https://github.com/mandiant/VM-Packages/issues/new?assignees=&labels=%3Anew%3A+package&projects=&template=new_package.yml&title=Package+proposal%3A+%3Cpackage_name%3E.vm):

> Tool Name: The name of the tool being installed (usually the file name with the .exe), normally different from the package name. For example fakenet-ng (package name) vs FakeNet-NG (tool name).

But I don't know this tool to be sure. @mandiant/commando-vm how should the tool executable be called?

Reply:
I'll extract this discussion to an issue so that we can merge this already.

Reply:
Extracted to https://github.com/mandiant/VM-Packages/issues/810